File: CPP/3.cpp
// Print all negative elements in an array.

#include <iostream>
#include <bits/stdc++.h>

std::vector<int>negative_num(const std::vector<int>& nums) {
    std::vector<int> ans;
    for(int i=0;i<nums.size();i++) {
        if(nums[i] < 0) {
            ans.push_back(nums[i]);
        }
    }
    return ans;
} 

int main() {
    int n;
    std::cin>>n;
    std::vector<int>ans(n);
    for(int i=0;i<ans.size();i++) {
        std::cin>>ans[i];
    }
    std::vector<int>invoke = negative_num(ans);
    for(int i=0;i<invoke.size();i++) {
        std::cout<<invoke[i] << " " << std::endl;
    }
    return 0;
}

File: Python/3.py
import os

def FindNegative(nums):
    negative = []
    for i in range(0, len(nums)):
        if nums[i] < 0:
            negative.append(nums[i])
    return negative

nums = list(map(int, input().split()))
invoke = FindNegative(nums)
print("Negative Numbers: ", invoke)


